pennylane/templates/subroutines/qsvt.py

```diff
@@ -953,11 +953,16 @@ def circuit_qsvt():
 
     """
 
+    poly = qml.math.array(poly, like="numpy")
```

Comment:
If all the interfaces are being unified to `numpy`, then why not simply use `qml.math.trim_zeros`?
```suggestion
    poly = qml.math.trim_zeros(qml.math.array(poly, like="numpy"))
```